subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_user_reset_action.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "fp_storage_manager.h"
+
+__weak int bt_fast_pair_factory_reset_user_action_perform(void)
```

Comment:
It doesn't build because of lack of `__weak` definition. Please add `#include <zephyr/kernel.h>` or `#include <zephyr/toolchain.h>`